### PR TITLE
Support path import

### PR DESCRIPTION
**Pull request by farhanlatheef**
- Ref: #2148 

**Description**
- Added: Allow path import statement eg: `import Button from 'neetoui/Button'`

**Checklist**

~- [ ] I have made corresponding changes to the documentation.~
- [x] I have updated the types definition of modified exports.
- [x] I have verified the functionality in some of the neeto web-apps.
~- [ ] I have added tests that prove my fix is effective or that my feature works.~
~- [ ] I have added proper `data-cy` and `data-testid` attributes.~
- [x] I have added the necessary label (`patch`/`minor`/`major` - If package publish
      is required).

**Note**
This PR need small change to product's `app/javascript/jsconfig.json`.
~"neetoui/\*": ["../../node_modules/@bigbinary/neetoui/*"]~
"neetoui/\*": ["../../node_modules/@bigbinary/neetoui/types/*"],

**Reviewers**

### CODE REVIEW

**Comment by farhanlatheef** [comment]
@josephmathew900 _a

File: package.json [hunk 1]
```diff
@@ -40,14 +41,24 @@
       "types": "./layouts.d.ts"
     },
     "./formik": {
-      "import": "./dist/formik.js",
-      "require": "./dist/cjs/formik.js",
+      "import": "./dist/formik/index.js",
+      "require": "./dist/cjs/formik/index.js",
```
Comment (josephmathew900) [inline package.json:45]: Screenshot 2024-04-09 at 10 37 31 AM

Generated file inside formik directory is `index.js.js`, but here we are expecting it to be `index.js`. Could you please check why it is being coming as `index.js.js`?"

**Review by josephmathew900 — COMMENTED** [review]
@farhanlatheef _a

**Comment by farhanlatheef** [comment]
@josephmathew900 _a

**Review by josephmathew900 — APPROVED** [review]
LGTM. @farhanlatheef _a

**Pull request merged**